Makefile.kokkos

```diff
@@ -1207,10 +1209,15 @@ ifeq ($(KOKKOS_INTERNAL_USE_ARCH_AMD_GFX940), 1)
   tmp := $(call kokkos_append_header,"$H""define KOKKOS_ARCH_AMD_GPU \"gfx940\"")
   KOKKOS_INTERNAL_AMD_ARCH_FLAG := --offload-arch=gfx940
 endif
-ifeq ($(KOKKOS_INTERNAL_USE_ARCH_AMD_GFX942), 1)
-  tmp := $(call kokkos_append_header,"$H""define KOKKOS_ARCH_AMD_GFX942")
-  tmp := $(call kokkos_append_header,"$H""define KOKKOS_ARCH_AMD_GPU \"gfx942\"")
-  KOKKOS_INTERNAL_AMD_ARCH_FLAG := --offload-arch=gfx942
+ifeq ($(kokkos_internal_use_arch_amd_gfx942), 1)
+  tmp := $(call kokkos_append_header,"$h""define kokkos_arch_amd_gfx942")
+  tmp := $(call kokkos_append_header,"$h""define kokkos_arch_amd_gpu \"gfx942\"")
```

Comment:
What happened here? Why is everything lowercase?

Reply:
ups no idea I was just not looking far enough but somehow the copy paste in vim made it all lower case ...

Reply:
fixed